scripts/test-build-config

```diff
@@ -156,7 +156,7 @@ if ! make; then
     exit 1
 fi
 
-# Finally run the tests in Simulator mode or on Hardware
+# Finally run the tests in simulation or on Hardware
```

Comment:
Something I have noticed is that throughout this documentation we switch between 'Simulator =/- mode' and then "Simulation +/- mode". To be potentially nitpicky, I would try to get a more uniform name. This can be applied to a few of the above occurrences.

Reply:
@BRMcLaren This was an artifact of the existing documentation. I have tried to clean it up (per comments above) to make it "Simulation". Please let me know where you are seeing "Simulator"